test/Rings/integer-localizations.jl

```diff
@@ -0,0 +1,276 @@
+#################################################################################
+# A minimal implementation of the localization interface for the ring ℤ         #
+#################################################################################
+
+using Oscar
```

Comment:
I don't think this should be in a test file. That makes re-running the tests after a code change awkward or even impossible.

You reported trouble earlier with putting your code into `examples`. Let's try resolve those instead of papering over them? I think @fieker probably knows best how they work and perhaps can provide assistance / information, as soon as you tell us what the exact problem is?

Reply:
I explicitly requested it to be move to tests, because it is a test for the interface, but not the localization of ZZ that should be accessible for the user. And please, do not add more stuff to the examples. I wish this folder would finally be purged, see https://github.com/oscar-system/Oscar.jl/issues/280.

Reply:
We already adressed these questions above in the discussion.